templates.dist/form/defaultform.php

```diff
@@ -65,16 +65,22 @@
 		if ( in_array( $input, array( 'message' ) ) ) {
 			$isRequiredMessage = $pForm->isRequiredField( 'message' );
 			$additionMessage   = $isRequiredMessage ? '*' : '';
-			echo $pForm->getFieldLabel( 'message' );
-			echo $additionMessage . ':<br>';
-			echo '<textarea name="message">' . $pForm->getFieldValue( 'message' ) . '</textarea><br>';
+			$isHiddenField = $pForm->isHiddenField('message');
+			if (!$isHiddenField) {
+				echo $pForm->getFieldLabel( 'message' );
+				echo $additionMessage . ':<br>';
+				echo '<textarea name="message">' . $pForm->getFieldValue('message') . '</textarea><br>';
+			} else {
+				echo '<textarea name="message" class="oo-hidden-field" disabled">' . $pForm->getFieldValue('message') . '</textarea>';
```

Comment:
There is a stray `"` double-quotation after `disabled` and please use `readonly` instead of `disabled`.